<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplexActionDialog</class>
 <widget class="QDialog" name="ComplexActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Новое составное действие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="ruleTreeWidget">
       <column>
        <property name="text">
         <string>Действия</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="orPushButton">
       <property name="text">
        <string> Альтернатива</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startGroupPushButton">
       <property name="text">
        <string>Начать группу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishGroupPushButton">
       <property name="text">
        <string>Завершить группу</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startSetPushButton">
       <property name="text">
        <string>Начать множество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishSetPushButton">
       <property name="text">
        <string>Завершить множество</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveScenarioPushButton">
       <property name="text">
        <string>Сохранить сценарий</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Базовые действия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="baseActionListWidget"/>
     </item>
     <item>
      <widget class="QTreeWidget" name="complexActionTreeWidget">
       <column>
        <property name="text">
         <string>Составные действия</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="scenarioTreeWidget">
       <column>
        <property name="text">
         <string>Сценарии</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
